app/views/orders/new.html.erb

```diff
@@ -12,9 +12,14 @@
   <% cart.items.each do |item, quantity| %>
     <tr id='order-item-<%= item.id%>'>
       <td><h2><%= link_to item.name, "/items/#{item.id}" %></h2></td>
-      <td><p><%= number_to_currency(item.price) %></p></td>
       <td><p><%= quantity %></p></td>
-      <td><p><%= number_to_currency(cart.subtotal(item)) %></p><td>
+      <% if !(item.no_discounts?(quantity))%>
+        <td><p><%= number_to_currency(item.apply_discount((item.select_highest_discount(quantity).percentage))) %></p></td>
```

Comment:
so if you see the note on apply_discount above, this is the issue i'm running into, having to call select_highest_discount and then grab that discounts percentage, not the cleanest solution